### ISSUE TITLE
Threads monitor pane

### ISSUE THREAD

**Issue by ykud** [bgregs514/vscode-tm1#5]
This is obviously a low priority.

It'd be great to be able to see the current threads on the server, similarly to a tm1top output somewhere on the bottom pane of the VSCode window.

There's quite a few examples of the REST API calls required, like [this one](https://blog.quebit.com/blog/quebit-blog-ibm-cognos-tm1-sdk-delivering-tm1top-data-with-postman-and-the-rest-api). 

Something along the lines of [this extension for visualisation](https://github.com/weinand/vscode-processes)?

**Comment by silw** [comment]
maybe editor window is better place for top?

**Comment by bgregs514** [comment]
Very cool idea!  Let's keep this one in mind for future releases.  As @silw mentioned, there may be a few different ways to present the data, but we can discuss further once we pull this into our development focus.

**Comment by ykud** [comment]
Yep, we'll figure all of this out when we get going on it. A 'view' in VScode can be placed anywhere :)
There's a few more of these, I'll see if I manage to create a TreeProvider with them as an overview and then splice it into issues.
